projects/plugins/jetpack/modules/infinite-scroll/infinity.php

```diff
@@ -1541,6 +1541,11 @@ public function query() {
 					sharing_register_post_for_share_counts( get_the_ID() );
 				}
 
+				// If sharing counts are not initialized for any reason, we initialize them here.
+				if ( ! isset( $jetpack_sharing_counts ) || ! is_array( $jetpack_sharing_counts ) ) {
```

Comment:
No need for the `isset` here, the `global` forces it to exist (with value `null`) if it didn't already.
```suggestion
				if ( ! is_array( $jetpack_sharing_counts ) ) {
```